bitnami/mongodb/templates/standalone/dep-sts.yaml

```diff
@@ -299,9 +299,9 @@ spec:
           args:
             - |
               {{- if .Values.auth.enabled }}
-              /bin/mongodb_exporter --mongodb.uri mongodb://root:$(echo $MONGODB_ROOT_PASSWORD | sed -r "s/@/%40/g;s/:/%3A/g")@localhost:27017/admin{{ .Values.metrics.extraUri }} {{ .Values.metrics.extraFlags }}
+              /bin/mongodb_exporter --mongodb.uri "mongodb://root:$(echo $MONGODB_ROOT_PASSWORD | sed -r "s/@/%40/g;s/:/%3A/g")@localhost:27017/admin{{ .Values.metrics.extraUri }}" {{ .Values.metrics.extraFlags }}
```

Comment:
I think we could build an uri base on the tls parameters automatically when `tls.enabled=true` and then append the `.Values.metrics.extraUri`, so that user won't have to be aware to add extra uri parameters when enabling tls, what do you think?

Reply:
That could work...how exactly would you suggest we set that up when tls.enabled=true ?

Reply:
I applied a change for when tls.enabled=true, let me know what you think.